_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+
+## Thank you to our alumni 💐
+
+Like in any other project, there naturally comes a time for some team members to move on to other challenges. Since our last update, we have bid farewell to Christian (@christianparpart), Damian (@wechman), Đorđe (@mijovic), Hari (@hrkrshnn), Mathias (@marenz),  and Nishant (@nishant-sachdeva). We were extremely glad to have them and wish them all the best for their future! Luckily, some alumni remain in our orbit and continue to contribute with code and input to language design discussions.
+
+## Introducing new team members 💝
+
+On a happy note, we are excited to introduce new core team members who have joined us along the way. Please meet Matheus, Nikola, Nuno and Rodrigo!
+
+In order to give you a chance to get to know them better, we asked all new joiners to respond to a selection of questions. Read their responses below. 
+
+### Matheus - @[matheusaaguiar](https://github.com/matheusaaguiar)
+
+#### What is your professional background?
+
+_I have a BSc in Computer Science and a MSc in Artificial Intelligence. After finishing college, I worked as a C++ backend developer on a small educational game project for about 2 years. Subsequently, I worked as a Data Structure and Algorithms teacher for 8 years. I joined the Solidity team in May 2022 and am working on the Solidity Compiler since._
+
+#### What made you join the Solidity team?
+
+_After being a teacher for a fair share of years, I wanted to work with the industry on new emerging technologies and challenge myself. Besides that, I always wanted to work on an open-source project. It is really exciting to be learning and building with such great devs and user community._
+
+#### What do you like to do in your free time?
+
+_I enjoy playing digital games such as ~~cracktorio~~ Factorio and StarCraft II, but also like to gather some friends and play board games while drinking some beers 🍻. Other than that, I really like skateboarding, running and football of course 🇧🇷!_
+
+
+### Nikola - @[nikola-matic](https://github.com/nikola-matic)
+
+#### What is your professional background?
+
+_I have a BSc in Comp Sci, and a ~10 year long obsession with C++, which kinda narrowed down my options as far as jobs were concerned, because no one's really gonna write a web app in C++. Having said that, I ended up working on a ton of interesting projects throughout my career, like writings high-perf server-client applications for SMS messaging, hardware virtualization on the cloud, EV routing, and a ton of other stuff. Unfortunately, up until I joined the EF and the Solidity team, I've never worked on an open source project, so this is a first for me (hopefully not the last)._
```

Comment:
```suggestion
_I have a BSc in Comp Sci and a ~10-year-long obsession with C++, which kinda narrowed down my options as far as jobs were concerned because no one's really gonna write a web app in C++. Having said that, I ended up working on a ton of interesting projects throughout my career, like writings high-perf server-client applications for SMS messaging, hardware virtualization on the cloud, EV routing, and a ton of other stuff. Unfortunately, up until I joined the EF and the Solidity team, I've never worked on an open-source project, so this is a first for me (hopefully not the last)._
```